go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/databricks/cli
 
-go 1.22
+go 1.22.0
```

Comment:
Please revert.

Reply:
I will. But it breaks my dev environment because it can't find toolchain for 1.22. Do you have this problem? I use IntelliJ

Reply:
Not with VS Code (nor command line). Does it maybe use a bundled Go toolchain?

Reply:
I couldn't find where toolchain is configured, but thanks for pointers, I will try to figure it out 👍